app/scripts/models/auth_brokers/fx-sync.js

```diff
@@ -3,255 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /**
- * A generic broker that can communicate with Firefox when used to sign in
- * to Sync. Concrete sub-classes should define the function `createChannel`
- * and a `commands` object.
+ * A generic broker used to define behaviors when integrating with Sync.
+ * Used as a base class for brokers that communicate with Firefox, and
```

Comment:
other comments mentioned this, but just so I don't lose this, should this say `communicate with Firefox Desktop`?